app/templates/all_questions/en/dpif_r2_all_questions_en.html

```diff
@@ -482,4 +482,4 @@ <h4 class="govuk-heading-s">
 </div>
     </div>
 </div>
-{% endblock %}
+{% endblock content %}
```

Comment:
same here ?

Reply:
yes, it is one of the linting errors, that we need to name the blocks.